README.md

```diff
@@ -34,6 +34,8 @@ Below is a list of versions of `openshift-ansible` the CASL team has leveraged a
 
 | casl-ansible version | OpenShift version | openshift-ansible version | openshift-ansible-contrib version | infra-ansible version |
 |:-------------------------:|:-----------------:|:-----------:|:------------:|:-------------:|
+| [v3.6.1](https://github.com/redhat-cop/casl-ansible/releases/tag/v3.6.1) | OCP 3.7.x | [3.7.11-1](https://github.com/openshift/openshift-ansible/archive/openshift-ansible-3.7.11-1.tar.gz) | [v3.6.0](https://github.com/openshift/openshift-ansible-contrib/releases/tag/v3.6.0) | master |
```

Comment:
I think I would prefer (for now) to just have this point to master, assuming it works, so as to avoid version confusion.

Reply:
or should we cut a 3.7 release real quick?